src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -41,4 +43,20 @@
         networkModificationService.changeSwitchState(networkUuid, switchId, open);
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping(value = "/networks/{networkUuid}/{equipmentType}/{equipmentId}")
+    @ApiOperation(value = "change a switch state in the network", produces = "application/json")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "The equipment state has been changed")})
+    public ResponseEntity<Map<String, Boolean>> changeEquipmentState(
+        @ApiParam(value = "Network UUID") @PathVariable("networkUuid") UUID networkUuid,
+        @ApiParam(value = "Equipment Type") @PathVariable("equipmentType") ModifiableEquipmentType equipmentType,
+        @ApiParam(value = "Equipment Type") @PathVariable("equipmentId") String equipmentId,
+        @RequestBody() Map<String, String> changeRequest) {
+        Map<String, Boolean> changes = networkModificationService.changEquipmentState(networkUuid, equipmentType, equipmentId, changeRequest);
+        if (!changes.isEmpty()) {
+            return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(changes);
+        }
+        return ResponseEntity.badRequest().build();
+    }
+
 }
```

Comment:
Maybe, we could handle the switch modification process with the same service ...
(to see later ....)